e2e/testdata/prometheus/prometheus.yaml

```diff
@@ -174,7 +174,7 @@ spec:
           resources:
             requests:
               cpu: 500m
-              memory: 500M
+              memory: 1Gi
             limits:
               cpu: 1
               memory: 1Gi
```

Comment:
Do you think we should bump the limit or leave as-is?

Reply:
If it's showing OOMKilled in the container status, it's probably because of the limit, not the request. Request might get it killed by the kernel oomkiller, but that usually shows up as a different failure.

Reply:
I think the current limit (1Gi) should be okay.

The failure only happens on the `autopilot-rapid-latest` test cluster. For Autopilot clusters, the limit is reset to be the same as request (adjusted), which is `512Mi`.